query.go

```diff
@@ -0,0 +1,78 @@
+package beacon
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"cloud.google.com/go/bigquery"
+)
+
```

Comment:
Exported types need commentary.  https://golang.org/doc/effective_go.html#commentary

// Query holds information about a single query against a Beacon.

or something like this

other exported names need commentary too

Reply:
done